Eigen/src/Geometry/EulerAngles.h

```diff
@@ -55,7 +55,12 @@ MatrixBase<Derived>::eulerAngles(Index a0, Index a1, Index a2) const
     res[0] = atan2(coeff(j,i), coeff(k,i));
     if((odd && res[0]<Scalar(0)) || ((!odd) && res[0]>Scalar(0)))
     {
-      res[0] = (res[0] > Scalar(0)) ? res[0] - Scalar(EIGEN_PI) : res[0] + Scalar(EIGEN_PI);
+      if(res[0] > Scalar(0)) {
+        res[0] -= Scalar(EIGEN_PI);
+      }
+      else {
+        res[0] += Scalar(EIGEN_PI);
+      }
```

Comment:
I believe this can be simplified to
```c++
if ((odd && res[0]<Scalar(0)) 
  res[0] += Scalar(EIGEN_PI);
else if (((!odd) && res[0]>Scalar(0)))
  res[0] -= Scalar(EIGEN_PI);
```